contracts/ISpotV1.sol

```diff
@@ -8,6 +8,17 @@ interface ISpotV1 {
         uint outMin;
     }
 
+    struct Log {
+        uint timestamp;
+        uint blocknumber;
+        uint8 operationType; // 0 - deposit, 1 - withdrawal, 2 - restake
```

Comment:
LOW: create exact event types for every event type, check this - https://medium.com/mycrypto/understanding-event-logs-on-the-ethereum-blockchain-f4ae7ba50378